dask/dataframe/utils.py

```diff
@@ -952,3 +952,14 @@ def valid_divisions(divisions):
         return False
 
     return True
+
+
+def drop_by_shallow_copy(df, columns):
+    """ Use shallow copy to drop columns in place
+    """
+    df2 = df.copy(deep=False)
+    if isinstance(columns, str):
```

Comment:
Pandas columns can also be tuples or numbers I think, so something like the following might be wiser:

```suggestion
    if not isinstance(columns, (list, pd.Index)):
```

Reply:
@TomAugspurger if you have a chance to look this over briefly your experience would be welcome.

Reply:
```suggestion
    if not pd.api.types.is_list_like(columns):
        columns = [columns]
```

Reply:
Thanks @TomAugspurger !